reference.md

```diff
@@ -927,3 +927,23 @@ Field Name | Presence | Type | Description
   }
 }
 ```
+
+
+### deeplink with query params
+
+The uris provided in `realtime_booking` (`android_uri`, `ios_uri`, `web_uri`) and `booking_rules` (`booking_url`) should support a specific set of query params:
+
+Param | Presence | Type | Description
+---|---|---|---
+`pickup_lat` | REQUIRED | Latitude | Latitude of the location where the user will be picked-up.
+`pickup_lon` | REQUIRED | Longitude | Longitude of the location where the user will be picked-up.
+`drop_off_lat` | REQUIRED | Latitude | Latitude of the location where the user will be dropped off.
+`drop_off_lon` | REQUIRED | Longitude | Longitude of the location where the user will be dropped off.
```

Comment:
```suggestion
`dropoff_lat` | REQUIRED | Latitude | Latitude of the location where the user will be dropped off.
`dropoff_lon` | REQUIRED | Longitude | Longitude of the location where the user will be dropped off.
```

Reply:
Shouldn't we keep the same names everywhere? We use `drop_off` for the `wait_time` and `booking_rules` endpoint.

Reply:
Agreed

Reply:
The "real" spelling (as much as a word spelling can be real) is "drop-off": https://www.merriam-webster.com/dictionary/drop-off.

Reply:
I stand corrected, not sure why I thought it was dropoff. It's the same thing in GTFS.